content/automate/ManageIQ/Transformation/Common.class/__methods__/acquiretransformationhost.rb

```diff
@@ -22,7 +22,7 @@ def main
               destination_ems = task.transformation_destination(source_cluster).ext_management_system
               raise "Invalid destination EMS type: #{destination_ems.emstype}. Aborting." unless destination_ems.emstype == "rhevm"
 
-              transformation_host = ManageIQ::Automate::Transformation::TransformationHosts::Common::Utils.get_transformation_host(destination_ems, 'vddk', factory_config)
+              transformation_host = ManageIQ::Automate::Transformation::TransformationHosts::Common::Utils.get_transformation_host(destination_ems, @handle.get_state_var(:transformation_method), factory_config)
```

Comment:
@fdupont-redhat 
Should the transformation_method be set in request/task options hash, since we want that information to persist. The state_var is temporary and will disappear when the task ends. The Request/Task options would persist.

Reply:
@mkanoor 
The transformation method is set in the AssessTransformation method. Currently, it's hardcoded to 'vddk', but in future versions 'ssh' will also be available and provided by the task or transformation plan, or even the transformation mapping. We still don't know which of these objects will hold the information.

Reply:
@fdupont-redhat 
Do you want to open a git issue on moving this to the request or task options instead of leaving it in state vars?

Reply:
Shouldn't I wait for this PR to be merged before ? If not, I'll open the issue.